### PR TITLE
feat: Add form and button to edit Batch Export

### PR DESCRIPTION
**Pull request by tomasfarias**
## Problem

Batch Exports cannot be edited via the UI... until now!

## Changes

This PR adds an edit button to the ViewExport scene that sends you over to the same form used for export creation, pre-populated with export values.

This also means that the CreateExport form has now been moved to ExportForm.tsx as it's used for both create and update. The new form looks a bit nicer now (stole things from the FeatureFlag form):

Screenshot 2023-08-02 at 14 31 46

👉 *Stay up-to-date with [PostHog coding conventions](https://posthog.com/docs/contribute/coding-conventions) for a smoother review.*

### CODE REVIEW

File: frontend/src/scenes/exports/ExportForm.tsx [hunk 1]
```diff
@@ -23,24 +30,66 @@ import { urls } from '../urls'
...
+    let exportLoading = false
+    let exportError = null
+    if (exportId) {
+        const { loading, export_, error } = useExport(currentTeamId, exportId)
+        existingExport = export_
+        exportLoading = loading
+        exportError = error
+    }
+    const [exportType, setExportType] = useState<'S3' | 'Snowflake'>(
+        existingExport ? existingExport.destination.type : 'S3'
```
Comment (tiina303) [inline frontend/src/scenes/exports/ExportForm.tsx:52]: nit: I'd not have a default export type

**Review by tiina303 — APPROVED** [review]
Could you make the form have a bit more details what empty start or end date mean, it's not obvious to users otherwise

**Comment by tomasfarias** [comment]
~Ugh, not sure why the test is failing.~ I did something to fix it.

**Comment by tomasfarias** [comment]
Closed in favor of #16926

**Pull request closed without merging**